packages/sdks-tests/src/specs/index.ts

```diff
@@ -74,110 +74,119 @@ function isBrowser(): boolean {
-  '/element-events': elementEvents,
-  '/external-data': EXTERNAL_DATA,
-  '/show-hide-if': SHOW_HIDE_IF,
-  '/show-hide-if-repeats': SHOW_HIDE_IF_REPEATS,
-  '/custom-breakpoints-reset': customBreakpointsReset,
-  '/text-block': textBlock,
-  '/text-eval': textEval,
-  '/state-binding': stateBinding,
-  '/nested-symbols': nestedSymbols,
-  '/personalization-container': personalizatContainer,
-  '/editing-styles': EDITING_STYLES,
-  '/video': video,
-  '/repeat-items-bindings': REPEAT_ITEMS_BINDINGS,
-  '/input-default-value': INPUT_DEFAULT_VALUE,
-  '/duplicate-attributes': DUPLICATE_ATTRIBUTES,
-  '/js-content-is-browser': JS_CONTENT_IS_BROWSER,
-  '/slot': SLOT,
-  '/slot-with-symbol': SLOT_WITH_SYMBOL,
-  '/slot-without-symbol': SLOT_WITHOUT_SYMBOL,
-  '/no-trusted-hosts': HOMEPAGE,
-  '/editing-styles-no-trusted-hosts': EDITING_STYLES,
-  '/animations': ANIMATIONS,
-  '/data-preview': DATA_PREVIEW,
-  '/form': FORM,
-  '/default-styles': DEFAULT_STYLES,
-  '/css-properties': CSS_PROPERTIES,
-  '/hover-animation': HOVER_ANIMATION,
-  '/tabs': TABS,
-  '/custom-components': CUSTOM_COMPONENTS,
-  '/basic-styles': BASIC_STYLES,
-  '/accordion': ACCORDION,
-  '/accordion-one-at-a-time': ACCORDION_ONE_AT_A_TIME,
-  '/accordion-grid': ACCORDION_GRID,
-  '/symbol-tracking': SYMBOL_TRACKING,
-  '/columns-with-different-widths': COLUMNS_WITH_DIFFERENT_WIDTHS,
-  '/custom-components-models-show': CUSTOM_COMPONENTS_MODELS_RESTRICTION,
-  '/custom-components-models-not-show': CUSTOM_COMPONENTS_MODELS_RESTRICTION,
-  '/editing-box-columns-inner-layout': EDITING_BOX_TO_COLUMN_INNER_LAYOUT,
-  '/with-fetch-options': HOMEPAGE,
-  '/symbol-with-repeat-input-binding': SYMBOL_WITH_REPEAT_INPUT_BINDING,
-  '/children-slot-placement': CUSTOM_COMPONENT_CHILDREN_SLOT_PLACEMENT,
-  '/dynamic-loading': DYNAMIC_LOADING_CUSTOM_COMPONENTS,
-  '/eager-dynamic-loading': EAGER_DYNAMIC_LOADING_CUSTOM_COMPONENTS,
-  '/ssr-binding': SSR_BINDING_CONTENT,
-  '/blocks-class-name': BLOCKS_CLASS_NAME,
-  '/duplicated-content-using-nested-symbols': DUPLICATED_CONTENT_USING_NESTED_SYMBOLS,
-  '/custom-components-nowrap': CUSTOM_COMPONENTS_NOWRAP,
-  '/override-base-url': HTTP_REQUESTS,
-} as const;
+type Page = {
+  content: BuilderContent;
+  /**
+   * The target SDKs that this page should be tested against. This is important because certain frameworks
+   * (like NextJS) will pre-render all possible pages on the server. If a test is not meant to work in a specific app,
+   * then that app will fail to build when attempting to pre-render the page.
+   *
+   * Defaults to `all`.
+   */
+  target?: 'gen1' | 'gen2' | 'all';
+  /**
+   * To test visual editing in Gen 1 SDK, we cannot provide a hardcoded JSON.
+   * Instead, we have to let the SDK fetch the data from the API and mock the
+   * network request instead.
+   * NOTE: This is why I can only test visual editing for the old SDK when using a non-SSR app
+   */
+  isVisualEditingTest?: boolean;
```

Comment:
@samijaber can we name this specifically targeted to gen1, something like `isGen1VisualEditingTest` or anything similar? As there are other visual editor tests for columns or symbols where we only have them for gen2

Reply:
mm sure. we can rename it to `isVisualEditingTest` once all the visual editing tests have been updated to run on gen1

Reply:
yes! thank you